api/api_test_suite.go

```diff
@@ -0,0 +1,206 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/cloudflare/cfssl/csr"
+	"github.com/cloudflare/cfssl/helpers/testsuite"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+)
```

Comment:
I think the order for the imports is supposed to be things from Go's standard library and then other libraries. So it should be like:

    import (
        "bytes"
        "encoding/json"
        "fmt"
        "io/ioutil"
        "net/http"
        "os"
        "os/exec"
        "strconv"
        "strings"

        "github.com/cloudflare/cfssl/csr"
        "github.com/cloudflare/cfssl/helpers/testsuite"
    )